src/middleware/authentication.middleware.ts

```diff
@@ -2,13 +2,23 @@ import { NextFunction, Request, Response } from "express";
 import { authMiddleware, AuthOptions } from "@companieshouse/web-security-node";
 import { env } from "../config";
 import { removeLangFromUrl } from "../utils/query";
-
+import { Urls, PrefixedUrls } from "../constants";
 export const authenticationMiddleware = (req: Request, res: Response, next: NextFunction) => {
     const CHS_URL = env.CHS_URL;
+    const RETURN_URL = getReturnCallbackURL(req);
```

Comment:
This local variable should be in camel case

Reply:
Fixed. Can you please review again?